fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java

```diff
@@ -230,7 +230,8 @@ public void setJob(ExportStmt stmt) throws UserException {
         this.parallelNum = stmt.getParallelNum();
         this.exportPath = path;
         this.sessionVariables = stmt.getSessionVariables();
-        this.timeoutSecond = sessionVariables.getQueryTimeoutS();
+        this.timeoutSecond = stmt.getTimeout();
+        this.sessionVariables.setQueryTimeoutS(this.timeoutSecond);
```

Comment:
here we should clone one session variables to avoid change origin session variable values

Reply:
+1

Reply:
done